PaTTAN Room Displays/PaTTAN Room Displays/NoMeetingsPage.xaml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             x:Class="PaTTAN_Room_Displays.NoMeetingsPage"
+             Title="NoMeetingsPage"
+             BackgroundColor="White"
+             >
+
+    <StackLayout>
+        <Grid RowSpacing="25" 
```

Comment:
Why is this a grid, if there will only be one cell?

Reply:
I don't know what Grid does yet.   Clearly we have much to discuss.